test/spec/znkExercise/znkExerciseDrv.spec.js

```diff
@@ -695,15 +695,15 @@ describe('testing directive "znkExerciseDrv":', function () {
         expect(content.hasClass('done-btn-show')).toBeFalsy();
     });
 
-    it('given view mode is review and initForceDoneBtnDisplay is true then only done-btn-show class should be added',function(){
+    it('given view mode is review and initForceDoneBtnDisplay is true then it still should hide the done button',function(){
```

Comment:
i think that review mode by definition is always after complete without done button, there's no point for done button if it's all complete. so i don't think it should be flexible. you think i should change it back?